db/docker-compose.yaml

```diff
@@ -21,4 +21,6 @@ services:
       "-user=gatehouse",
       "-password=gatehouse",
       "-locations=filesystem:./migrations"
-    ]
\ No newline at end of file
+    ]
+    environment:
+      JAVA_ARGS: -XX:UseSVE=0 -XX:+IgnoreUnrecognizedVMOptions
```

Comment:
Flyby fix, Docker (atleast Colima) is struggling to run Java images right now on our new macbooks: https://github.com/abiosoft/colima/issues/965

The `UseSVE` flag only exists on Arm versions of Java, hence also requiring `IgnoreUnrecognizedVMOptions` to allow Java to run on the x86_64 Github Actions runner.